imgui/core.pyx

```diff
@@ -259,7 +259,7 @@ CONFIG_NAV_ENABLE_GAMEPAD = enums.ImGuiConfigFlags_.ImGuiConfigFlags_NavEnableGa
 CONFIG_NAV_ENABLE_SET_MOUSE_POS = enums.ImGuiConfigFlags_.ImGuiConfigFlags_NavEnableSetMousePos
 CONFIG_NAV_NO_CAPTURE_KEYBOARD = enums.ImGuiConfigFlags_.ImGuiConfigFlags_NavNoCaptureKeyboard
 CONFIG_NO_MOUSE = enums.ImGuiConfigFlags_.ImGuiConfigFlags_NoMouse
-CONFIG_NO_MOUSE_CURSOR_CHARGE = enums.ImGuiConfigFlags_.ImGuiConfigFlags_NoMouseCursorChange
+CONFIG_NO_MOUSE_CURSOR_CHANGE = enums.ImGuiConfigFlags_.ImGuiConfigFlags_NoMouseCursorChange
```

Comment:
This var is used in `integrations/pyglet.py`

Reply:
Then either pyglet.py should also be changed, or (if backward compatibility with wrong interface is preferred, leave the the older code in (so both CHANGE and CHARGE have the same value.)

Reply:
Considering you are changing it in this file, I would say it's supposed to be part of this PR as well,
otherwise you'll leave this repository in a broken state.

Reply:
Love to help, but I'm not great with git/github integrations. Sorry if I'm making this more difficult than it should be.

 I don't see CONFIG_NO_MOUSE_CURSOR_CHARGE (or CHANGE) in integrations/pyglet.py, on any branch, so not sure how/where I can make that change.

Reply:
Sorry, that was a local change I made on my repo.